.travis.yml

```diff
@@ -10,9 +10,9 @@ before_install:
   - sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
   - sudo apt-get update
   - sudo apt-get -y install docker-ce
-  - docker pull docker.elastic.co/elasticsearch/elasticsearch:6.4.2
+  - docker pull docker.elastic.co/elasticsearch/elasticsearch:7.5.0
```

Comment:
there is 7.5.2 as well.

Reply:
I did not change the pipeline configuration since we will be immigrating to Github actions. We will update the versions once we immigrate.